buildSrc/src/main/kotlin/economy-java.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    `java-library`
+}
+
+group = "com.eternalcode"
+version = "1.0.0"
+
+java {
+    toolchain.languageVersion.set(JavaLanguageVersion.of(21))
+}
+
+tasks.compileJava {
+    options.compilerArgs = listOf("-Xlint:deprecation", "-parameters")
+    options.encoding = "UTF-8"
+    options.release = 17
```

Comment:
When we want to use runPaper on 1.21.1, it won't work if the gradle runtime runs on a version lower than 21, because Mojang in version 1.20.5 moved to Java 21, the plugin is compatible with java 17+, but gradle uses java 21.